lua/autorun/server/bounties.lua

```diff
@@ -46,7 +44,7 @@ RegisterChatCommand({'bounty','setbounty'},function(ply,arg)
 	local t = string.Explode(" ",arg)
 	local p = tonumber(table.remove(t))
 	local to,c = PlyCount(string.Implode(t))
```

Comment:
the seperator should be " ", the default is ""

Reply:
my bad, I forgot it needed the argument